components/button/src/dropdown-button/dropdown-button.js

```diff
@@ -67,7 +68,18 @@ class DropdownButton extends Component {
     }
     anchorRef = React.createRef()
 
-    onToggle = ({ name, value }, event) => {
+    controlledOnClick = ({ name, value } = {}, event) => {
```

Comment:
I've had to add a default value for the first argument here. To prevent the error in the screenshot below. It seems to only be needed by the Jest tests though, I could replicate this behaviour in the story.

In theory, when clicking the backdrop, the first arg (`payload`) should be `{}` ([see `Layer`](https://github.com/dhis2/ui/blob/afc6eb5ef84a9796330b7eeac8addd2dd337f786/components/layer/src/layer.js#L9), but for some reason in the Jest test this is not true.
<img width="1048" alt="Screenshot 2021-11-29 at 13 54 21" src="https://user-images.githubusercontent.com/353236/143872999-ef45680a-2681-433a-9dc0-fe2b4d08596c.png">

So while this default arg value fixes the problem, I don't fully understand the reason this problem occurred in the first place, which is slightly unnerving.

I just wanted to point this out.

Reply:
Solved in dcd9098